### ISSUE TITLE
TryValidateModel is not working for nested attributes

### ISSUE THREAD

**Issue by ThumNet** [shibayan/azure-functions-http-api#12]
Hi Shibayan,

When using `TryValidateModel` method DataAnnotations attributes (for example [Required]) on nested objects/properties  are not validated.

```csharp
public class SampleModel
{
    [Required]
    public string Name { get; set; }

    public string[] Array { get; set; }

    public NestedModel Nested { get; set; }
}

public class NestedModel
{
    [Required]
    public string Name { get; set; }
}
```

When posting the SampleModel and omitting the Name property, TryValidateModel returns `true` and not `false` as expected.

Tried with:
```json
{
    "name": "test",
    "nested":{       
    }
}
```
and also with:

```json
{
    "name": "test"
}
```

This is what I expected:
![image](https://user-images.githubusercontent.com/692308/125276810-f25b8c80-e310-11eb-99b0-4f66e5fcaddc.png)

Is this something you are aware of or a bug?

Ps I found a fix for the behavior described above, and if you are interested I can provide a PR.

**Comment by shibayan** [comment]
Thanks for the report. This issue has been fixed in `v1.1.1`.
Since the validation process is now common to ASP.NET Core, the behavior will be more consistent.

**Comment by ThumNet** [comment]
Wow, what a quick fix :)

Thank you!
